package.json

```
@@ -90,7 +90,9 @@
   },
   "lint-staged": {
     "*.{js,css,md,json}": [
-      "prettier --write",
+      "prettier --write"
+    ],
+    "*.{js,md,json}": [
```

If that is the case, I have another suggestion remove `md`

```suggestion
    "*.{js,json}": [
```

---

Ok I will remove it .
By the way I had discussed about this error with one more person in detail. You can have a look at it here https://github.com/EddieHubCommunity/LinkFree/issues/912

---

Hey @krishguptadev, I have made the suggested change.